### PR TITLE
Exposed toplevel status as a property for spatial and canvasitem

### PR DESCRIPTION
**Pull request by Duroxxigar**
Addresses #10453

### CODE REVIEW

File: doc/classes/CanvasItem.xml [hunk 1]
```diff
@@ -557,6 +557,9 @@
 		<member name="show_behind_parent" type="bool" setter="set_draw_behind_parent" getter="is_draw_behind_parent_enabled" default="false">
 			If [code]true[/code], the object draws behind its parent.
 		</member>
+		<member name="toplevel" type="bool" setter="set_as_toplevel" getter="is_set_as_toplevel">
+			If true, the node will not inherit its transform from parent [CanvasItem]s.
```
Comment (Xrayez) [inline doc/classes/CanvasItem.xml]: "true" should be wrapped as `[code]true[/code]` the way it's done for other descriptions for consistency, same for other changes.

Comment (Duroxxigar) [reply doc/classes/CanvasItem.xml]: Fixed

**Comment by willnationsdev** [comment]
Since it was last updated in February, I'd recommend rebasing this pull request. Or, if unavailable, mentioning in the related Issue that someone else should handle the PR.

**Comment by akien-mga** [comment]
We actually discussed this in a live Q&A today, exposing this as a property makes sense. It needs a rebase before it can be merged.

Might be worth mentioning in the docs how child nodes of a "toplevel" node behave if they are not toplevel themselves (see #24154).

**Comment by Duroxxigar** [comment]
uhhh - oops. This is why you don't work on stuff so late in the night I guess. Really don't feel like fixing this PR.

Closing and just created a new one - #42448

**Pull request closed without merging**